python/pyspark/sql/readwriter.py

```diff
@@ -742,10 +743,16 @@ def save(self, path=None, format=None, mode=None, partitionBy=None, **options):
     def insertInto(self, tableName, overwrite=None):
         """Inserts the content of the :class:`DataFrame` to the specified table.
 
+        Optionally overwriting any existing data.
```

Comment:
Can you convert this to `Parameters`? You can just write down the description like, this is equivalent with `DataFrameWrtier.mode("overwrite")`

Reply:
Done